common/conf.py

```diff
@@ -155,10 +155,10 @@ def get_previous_version(majornumber, minornumber=0):
     'community' : (oo_root + '/support/%s', ''),
     'omero' : (oo_root + '/omero/%s', ''),
     'bf' : (oo_root + '/bio-formats/%s', ''),
-    # One branch only doc links. Branched docs links in conf.py files for
-    # individual doc sets
-    'model_doc' : (docs_root + '/ome-model/5.5.7/%s', ''),
+    # Doc links
+    'model_doc' : (docs_root + '/latest/ome-model/%s', ''),
     'devs_doc' : (docs_root + '/contributing/%s', ''),
+    'bf_doc' : (docs_root + '/latest/bio-formats/%s', ''),
```

Comment:
Is this used in the contributing documentation only or in the OMERO documentation? In the former, this makes sense as the contributing documentation largely documents our latest software. For OMERO, arguably, we should be able to point at the documentation for the specific version of Bio-Formats shipped with OMERO.

Reply:
It's in both but I can make the OMERO one version specific if we prefer.

Reply:
That would mean we'd need to update it regularly though.

Reply:
Agreed. The main reason for hardcoding from my side is that https://github.com/openmicroscopy/ome-documentation/blob/v5.4.0-m3/omero/users/cli/import.rst refers to supported formats table which will be tightly coupled to the the version included into OMERO. There is part of the autogeneration scripts which reads properties from the OMERO version (patch versions), should we consider adding the bf_version there as well to make sure it's updated via the auto-generation PR?